asrtoolkit/align_json.py

```diff
@@ -34,29 +40,9 @@ def align_json(ref_txt, json_file, filename=None):
     output.write(filename)
 
 
-def main():
-    """
-    CLI for forced alignment tools
-    """
-    parser = argparse.ArgumentParser(
-        description="align a gk json file against a reference text file")
-    parser.add_argument("input_json",
-                        metavar="input_json",
-                        type=str,
-                        help="input gk json file")
-    parser.add_argument("ref",
-                        metavar="ref",
-                        type=str,
-                        help="reference text file")
-    parser.add_argument("output_filename",
-                        metavar="output_filename",
-                        type=str,
-                        default=None,
-                        help="output_filename")
-
-    args = parser.parse_args()
-    align_json(args.ref, args.input_json, args.output_filename)
+def cli():
+    Fire(align_json)
 
 
 if __name__ == "__main__":
-    main()
+    cli()
```

Comment:
Any reason for the extra function invocation?
```suggestion
    Fire(align_json)
```

Reply:
Idea behind that is to have a single source of truth for what the entry point is